orderbook/src/api/get_fee_and_quote.rs

```diff
@@ -1,26 +1,19 @@
-use crate::fee::MinFeeCalculating;
-use anyhow::{anyhow, Result};
-use chrono::{DateTime, Utc};
+use crate::api::post_quote::{OrderQuoteRequest, OrderQuoteSide, SellAmount};
+use crate::{
+    api::post_quote::{Fee, FeeError},
+    fee::MinFeeCalculating,
+};
+use anyhow::Result;
 use ethcontract::{H160, U256};
-use model::h160_hexadecimal;
-use model::{order::OrderKind, u256_decimal};
+use model::{h160_hexadecimal, u256_decimal};
 use serde::{Deserialize, Serialize};
-use shared::price_estimation::{self, PriceEstimating, PriceEstimationError};
-use std::convert::Infallible;
-use std::sync::Arc;
+use shared::price_estimation::PriceEstimating;
+use std::{convert::Infallible, sync::Arc};
 use warp::{hyper::StatusCode, reply, Filter, Rejection, Reply};
 
-#[derive(Serialize)]
-#[serde(rename_all = "camelCase")]
-struct Fee {
-    #[serde(with = "u256_decimal")]
-    amount: U256,
-    expiration_date: DateTime<Utc>,
-}
-
 #[derive(Deserialize)]
 #[serde(rename_all = "camelCase")]
-struct SellQuery {
+pub struct SellQuery {
```

Comment:
Does this still need to be `pub`?

Reply:
prolly not... Will try and go through things that were made public and fix it.